util/fipstools/acvp/acvptool/katemitter/emitter.go

```diff
@@ -0,0 +1,87 @@
+// katemitter is a package which (with best effort) will attempt to emit test case data to a configured file
+// for use by AWS-LC's file-based test framework. "Attempt" because it doesn't check for errors on the writer
+// destination. But this is mostly a facility to aid in getting KATs for algorithms and re-using the inputs/outputs
+// from the ACVP server after they have been validated as accurate.
```

Comment:
Is this converting the ACVP demo vectors into the file test format? Why not just run these tests through the ACVP module like the other KAT ZIPs?

Reply:
Ahhhh, are you planning to add the ACVP test vectors in a follow up PR or this one?

Reply:
Added them to this one in the recent push.